### DISCUSSION TITLE
<Image /> pre-loading pixel threshold

### DISCUSSION THREAD

**Discussion by paulgrieselhuber** [vercel/next.js#20032] [Help]
I am testing out the new `<Image />` component on an application which involves many images in a row. The lazy load functionality of `<Image />` is appealing, but given the importance of a "smooth" experience for this app (i.e. not waiting for an image to load while scrolling), we need to build the image list with the ability to prefetch images while they are still several "frames" out of "frame".

Thus far, my experience has been that this component does not load the image until it is in frame. For most applications this would be great, but we _need_ preloading ahead of the image being in frame.

Thus far, [react-lazy-load-image-component](https://www.npmjs.com/package/react-lazy-load-image-component) seems to do the trick quite well, but I would prefer to use the native Next.js component if possible.

Can `Image` be configured to handle this, or should we continue on with `react-lazy-load-image-component`?

**Comment by ivoilic** [comment]
In the [docs](https://nextjs.org/docs/api-reference/next/image#loading) for the image component you can set the loading property to `eager` which will cause an image to load immediately. For your use case you could set this property based on images that are about to be in view in order to preload them.

**Reply by AlexandraKlein** [reply]
What is the difference between `priority: true` and `loading: "eager"` ?
